extendr-macros/src/extendr_impl.rs

```diff
@@ -187,86 +191,45 @@ pub fn extendr_impl(mut item_impl: ItemImpl, opts: &ExtendrOptions) -> syn::Resu
     let meta_name = format_ident!("{}{}", wrappers::META_PREFIX, self_ty_name);
     let finalizer_name = format_ident!("__finalize__{}", self_ty_name);
 
-    let conversion_impls = if opts.use_try_from {
-        quote! {
-            // Output conversion function for this type.
-
-            impl TryFrom<Robj> for &#self_ty {
-                type Error = Error;
-
-                fn try_from(robj: Robj) -> Result<Self> {
-                    Self::try_from(&robj)
-                }
-            }
+    let conversion_impls = quote! {
+        // Output conversion function for this type.
 
-            impl TryFrom<Robj> for &mut #self_ty {
-                type Error = Error;
```

Comment:
Why do we remove the `mut` here? Is this handled by the recent PR we made where we allow mut?

Reply:
this is not a valid change. I hope it is no longer part of this PR.

Reply:
It was just moved. Mis leading diff